packages/vite-plugin-cloudflare/src/runner-worker/index.ts

```diff
@@ -173,7 +173,16 @@ export function createWorkerEntrypointWrapper(
 
 				if (url.pathname === INIT_PATH) {
 					const { 0: client, 1: server } = new WebSocketPair();
-					createModuleRunner(this.env, server);
+					try {
+						await createModuleRunner(this.env, server);
+					} catch (e) {
+						return new Response(
+							JSON.stringify(
+								e instanceof Error ? e.message : JSON.stringify(e)
+							),
```

Comment:
Is it intentional to stringify twice if the error is not an instance of `Error` here?

Reply:
no, sorry my bad, thanks for spotting it 🙏